### DISCUSSION TITLE
is RTMP blocked?

### DISCUSSION THREAD

**Discussion by srice777** [AdguardTeam/AdGuardHome#3499] [Q&A]
I set this up at our church, and coincidentally, our livestream started having issues. It could be purely coincidental, or it could be AdGuard. We stream to rtmp servers... which are something like this: rtmp://server.name/subdomain  These are not normal web domains... so I'm not sure if they run through dns servers or not.  Is AdGuard interfering with rtmp connections? How can I be sure? Thanks.

**Comment by ameshkov** [comment]
AdGuard Home does not mess with RTMP, but maybe the DNS server you use as an upstream have some issues with the RTMP server you're using? For instance, it may be resolving it to some IP in a different geo. Try changing the upstream server and see if it changes anything.

**Reply by srice777** [reply]
I'm using the same DNS server as I was before implementing AdGuard. So, that shouldn't be an issue.  I did confirm the stream service had issues. So I'm convinced it was coincidental. It is nice to know that AdGuard will not interfere with RTMP though. Thanks for the info.
